.gitlab-ci.yml

```diff
@@ -979,7 +982,7 @@ workflow:
   - when: manual
     allow_failure: true
 
-.on_system_probe_changes_or_manual:
+.on_system_probe_e2e_changes_or_manual:
```

Comment:
Maybe `.on_system_probe_and_e2e_changes_or_manual`?

Reply:
Ok ! Will change in a follow up !

Reply:
Well will do it now, I forgot to merge main

Reply:
Actually it should be `on_system_probe_or_e2e_changes_or_manual`